src/test/resources/pages/studentProfilePictureModalDefault.html

```diff
@@ -1,45 +1,41 @@
-<div style="display: block;" class="modal fade in" id="studentPhotoUploader" role="dialog" aria-labelledby="myModalLabel" aria-hidden="false">
-    <div class="modal-dialog modal-lg">
-        <div class="modal-content">
+   <div aria-hidden="false" aria-labelledby="myModalLabel" class="modal fade in" id="studentPhotoUploader" role="dialog" style="display: block;">
```

Comment:
Hey btw, I am going to be regenerating this file in #4498 so, is it possible for you to reverse this change?